src/fluree/server/handler.clj

```diff
@@ -255,38 +256,74 @@
           fuel 1000] ; TODO: get this for real
       (assoc-in resp [:headers "x-fdb-fuel"] (str fuel)))))
 
-(defn wrap-policy-metadata
-  "Both Fluree-Policy-Class and Fluree-Policy-Identity can be optionally
-  set in the header for non-credentialed queries/transactions. The primary
-  motivation is enforcing policy with SPARQL which doesn't have an opts
-  map option, although if set, it will also override policyClass/Identity
-  that might have otherwise been set in FlureeQL/JSON."
+(defn wrap-header-opts
+  "Extract options from headers, parse and validate them where necessary, and attach to
+  request. Opts set in the header override those specified within the transaction or
+  query. Intended mainly to provide a method of specifying opts for SPARQL queries"
```

Comment:
Super minor, but I would drop this last sentence since these opts apply generally.